tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,64 +1,133 @@
 <div class="dummy-notification-container">
   {{#each notifications as |notification|}}
-      {{notification-message notification=notification}}
+    {{notification-message notification=notification}}
   {{/each}}
 </div>
-<h2>frost-text</h2>
 
-<div id="content">
-  <div class="dummy-header">Basic</div>
-  <div style="padding: 10px 20px;">
-    {{frost-text
-      autofocus=true
-      placeholder='basic field'
-    }}
-  </div>
+<div class="dummy-body">
+    <div class="section">
+        <div class="title">frost-text</div>
+        <div class="example">
+            <div class="title">Basic</div>
+            <div class="snippet">
+              {{format-markdown "```handlebars
+{{frost-text
+  autofocus=true
+  placeholder='basic field'
+}}
+```"
+              }}
+            </div>
+            <div class="demo">
+              {{frost-text
+              autofocus=true
+              placeholder='basic field'
+              }}
+            </div>
+        </div>
 
-  <div class="dummy-header">Action</div>
-  <div style="padding: 10px 20px;">
-    {{frost-text
-      on-input=(action "text")
-    }}
-  </div>
+        <div class="example">
+            <div class="title">Action</div>
+            <div class="snippet">
+              {{format-markdown "```handlebars
+{{frost-text
+  onInput=(action 'text')
+}}
+```"
+              }}
+            </div>
+            <div class="demo">
+              {{frost-text
+              onInput=(action "text")
+              }}
+            </div>
+        </div>
 
-  <div class="dummy-header">Bound</div>
-  <div style="padding: 10px 20px;">
-    {{frost-text
-      on-input=(action "bind")
-      value=boundText
-    }}
-    <br>Bound Text: {{boundText}}<br>
-    {{frost-text
-      on-input=(action "bind2")
-      value=boundText
-    }}
-  </div>
+        <div class="example">
+            <div class="title">Bound</div>
+            <div class="snippet">
+              {{format-markdown "```handlebars
+{{frost-text
+  onInput=(action 'bind')
```

Comment:
I think to make these examples clearer you should use the same action name in each (e.g. 'handleInputEvent')